gen/templates/factory/bobfactory_main.bob_test.go.tpl

```diff
@@ -22,7 +22,7 @@ func TestCreate{{$tAlias.UpSingular}}(t *testing.T) {
     }
   }()
 
-  if _, err := New().New{{$tAlias.UpSingular}}(ctx).Create(ctx, tx); err != nil {
+  if _, err := New().New{{$tAlias.UpSingular}}WithContext(ctx).Create(ctx, tx); err != nil {
```

Comment:
Should the tests call `.New` or `.NewWithContext`? I think the former. Even though a ctx is available, passing it to the builder feels redundant.

Reply:
If a context is available, let's use it.

Actually, since Bob targets Go 1.24 at minimum, we should use [`t.Context()`](https://pkg.go.dev/testing#T.Context) instead of `context.Background()`

Reply:
Alright, I'll change `context.Background()` to `t.Context()` too.